about.html

```diff
@@ -12,53 +12,100 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
 
-    <img src="img/about-plan.png" alt="strategy">
+    <div class="container about-page">
+
+      <section class="top-content">
+
+        <header>
+          <nav>
+            <div class="logo">
+                <img src="img/lambda-black.png" alt="Lambda">
+            </div>
+            <div class="links">
+            <a href="../web-Sprint-Challenge-User-Interface-and-Git/index.html">Home</a>
+            <a href="../web-Sprint-Challenge-User-Interface-and-Git/about.html">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="#">Contact</a>
+        </div>
+        </nav>
     
-    Strategy
+            <div class="head free">
+              <!-- <img src="img/jumbo-about.png" alt="why" class="free"> -->
+                <!-- blank filler for background image -->
+            </div>
+            <section class="ab">
+
+              <h2>About</h2>
+       
+              <p> Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+             </p>
+           </section> 
+        </header>
+      
+      </section>
+     
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
+<div class="learnmore">
+
+  <div class="middle-content star strategy">
+      <img src="img/about-plan.png" alt="strategy">
+    
+       <h2>Strategy </h2>
+    
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        </p>
 
+    <button class="button">Learn More </button>
 
+  </div>
 
```

Comment:
For each of these sections, you can reuse a class to format each section instead of giving each section its own classname.  The only downside to this is a little bit of readability.